oneapp-server/dao/TanfGaHeaderDao.js

```diff
@@ -0,0 +1,62 @@
+const { SQLDataSource } = require('datasource-sql');
+const oracledb = require('oracledb');
+const { OneAppError } = require('../utils/OneAppError');
+
+const parseBooleanField = field => {
+  if (field === null) {
+    return field;
+  }
+  return field === 'Yes' ? true : false;
```

Comment:
the db is returning the description instead of the code here, which is dependent on the current language. ideas on how to address this? am thinking we'll need to check the language and query the code detail description?

Reply:
Haha, lovely. I think it makes the most sense (unless @mikepif needs the boolean values) to change up the GraphQL type to return a string and let the translated values pass through to the front-end.

Reply:
done :)